docs/redirects/filesystems.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "File Systems"
 layout: redirect
-redirect: /ops/filesystems.html
-permalink: /ops/filesystems/index.html
+redirect: /ops/filesystems/index.html
```

Comment:
The filesystem link changes are unrelated to the original issue introducing the other problems. I don't see a problem in fixing these in the same commit, but please double-check.